Update sdk

CedrikNikita opened the pull request:
Might be not the final state will retest it afterwards again
- also fixes issues with tokens loading and new middleware related

Liubov-crypto commented:
Claim tip - not working: 
https://user-images.githubusercontent.com/69896204/164741042-4ce9f1ce-ebc4-4f72-8799-cd9c6a971f70.mov

The chain name is not displayed when entering the wallet  #1402 
The name disappeared in a while after it was put by default.

I can't connect the web wallet with SH feed. 

Web wallet:
Create a new wallet - pass
Verify Seed phrase-pass
Change tokens/currency - pass
Send ae - pass
Receive ae - pass
Send FT - pass - disappeared from the wallet
Receive FT - pass disappeared from the wallet
Balances view- pass
Send via scan QR - pass
AE is displayed in Tx History (All/Sent/Received) - pass
Send Tip - pass 
Claim tip - failed
Tx history - pass
Tx details - pass
Invite link: create - pass
top up -pass
claim back - pass
Register a name - pass   
Set name pointer - pass
Set as default name - pass
Extend - pass
Settings - pass
Networks - pass

No automatic connection between SH feed and extension. 

FF/Chrome Extensions:
Create a new wallet - pass
Verify Seed phrase-pass
Change currency - pass
Send ae - pass
Receive ae - pass
Balances view- pass
Send via scan QR - pass
AE is displayed in Tx History (All/Sent/Received) - pass
Send Tip - pass
Send Tip via Twitter tip button - pass
Claim tip - failed
Tx history - pass
Tx details - pass
Invite link: create - pass
top up -pass
claim back - pass
Register a name - pass   
Set name pointer - pass
Set as default name - pass
Extend - pass
Settings - pass
Networks - pass

Sometimes the transaction history is not fully loaded:
![тр](https://user-images.githubusercontent.com/69896204/164756978-c501266b-6416-4532-8111-eed50346ecee.png)

CedrikNikita commented:
> The chain name is not displayed when entering the wallet #1402 The name disappeared in a while after it was put by default.

> I can't connect the web wallet with SH feed.

- Fixed issue with name is not displayed when entering the wallet
- fixed connection between wallet and SH feed
- did not reproduce issue about changing name from default to regular

Liubov-crypto commented:
I can confirm that: 
The chain name is not displayed when entering the wallet https://github.com/aeternity/superhero-wallet/issues/1402
The name disappeared in a while after it was put by default.
![name](https://user-images.githubusercontent.com/69896204/165952706-91f8a5c3-b561-48a6-b7fc-07559046b5ba.png)

I can't connect the web wallet with the SH feed. 
Claim tip in Web wallet - not working:
I used commit[ c7e66d4] / v.0.9.7 for testing.

kenodressel commented:
- I think the name issues might come from the middleware: https://mainnet.aeternity.io/mdw/aex9/by_name gives me an "internal server error"
- claiming tips work for me
- connecting to the frontend works as well
